guides/V1_wwsg_Webhooks.md

```diff
@@ -30,6 +30,8 @@ Available events are:
 
 **message-created**: each time a new message is created in the space
 
+**message-deleted**: each time a message is deleted from the space
```

Comment:
This aspect of the API is currently classified as `experimental`.  Typically we note for GraphQL calls this at the top, but in this case, this is an aspect of an existing API.  I would use a sentence indicating so such as:

> Please note that the **message-deleted** event API is currently `experimental` and subject to change.  It is being included here to engage developers early with this API

@jarusso FYI